app/pages/cif/project-revision/[projectRevision]/form/[formIndex]/index.tsx

```diff
@@ -35,23 +35,35 @@ const pageQuery = graphql`
             id
           }
         }
-        ...ProjectForm_projectRevision
         ...TaskList_projectRevision
+        ...ProjectForm_projectRevision
         ...ProjectFormSummary_projectRevision
+        ...ProjectContactForm_projectRevision
+        ...ProjectContactFormSummary_projectRevision
+        ...ProjectManagerFormGroup_projectRevision
+        ...ProjectManagerFormSummary_projectRevision
+        ...ProjectQuarterlyReportForm_projectRevision
+        ...ProjectAnnualReportForm_projectRevision
```

Comment:
I'm wondering if eventually, we can wrap this component in a factory that allows us to pass just the query we need instead of the entire list of fragments